### DISCUSSION TITLE
Hyper link error

### DISCUSSION THREAD

**Discussion by kahaboom** [rfennell/AzurePipelines#1430] [Q&A]
Hello @rfennell and community. 

I am trying to set a hyper link to the build pipeline, but it's returning me to the wiki page.

I am using this  -  `## Build [{{buildDetails.buildNumber}}]({{buildid.url}})`

But it's returning me to the same wiki page from where I am pressing the link.

<img width="747" alt="Screenshot 2022-12-08 at 15 25 19" src="https://user-images.githubusercontent.com/80584072/206457764-c1c3e9b3-aead-4cb4-ad4a-0b8fcd148ece.png">
<img width="407" alt="Screenshot 2022-12-08 at 15 25 41" src="https://user-images.githubusercontent.com/80584072/206457839-953abd9b-a167-4de9-aff5-f65f982886fc.png">

Can somebody help why it's not working ?

**Comment by kahaboom** [comment]
So like when I am pressing "11", it's returning me to the same wiki page (as in screenshot) and not to the build pipeline summary

**Comment by rfennell** [comment]
The issue is the URL returned in the dataset directly is not the exact you you need

For example for WI we get a URL that needs to be revised to go to the edit form. This is done with a string replace.

```
{{#forEach this.Workitems}}
    - [{{this.id}}]({{replace this.url "_apis/wit/workItems" "_workitems/edit"}}) - {{lookup this.fields 'System.Title'}}
{{/forEach}}
```

In your case I would 

1. Out the `builddetails.url` as test
2. Look to see how it needs to be updated to the correct url for your needs
3. Update your handlerbars template to write the correct markdown
